plugins/modules/rma_workflow_manager.py

```diff
@@ -865,16 +867,48 @@ def unmark_device_for_replacement(self):
                 "Error while unmarking device for replacement"
             )
             self.status = task_result["status"]
-            self.msg = task_result["msg"]
+            self.msg = "RMA failed to replace the device: {0}".format(task_result["msg"])
             if self.status == "success":
                 self.result['changed'] = True
 
-        except Exception as e:
+        except Exception:
             self.status = "failed"
-            self.msg = "Exception occurred while unmarking device for replacement: {0}".format(str(e))
+            self.msg = "RMA failed to replace the device: No device found for unmarking replacement"
             self.log(self.msg, "ERROR")
         return self
 
+    def return_replacement_devices(self):
```

Comment:
get_device_id_ready_for_replacement. (or) get_ready_for_replacement_device_id  ??

Reply:
update the code accordingly with get_ready_for_replacement_device_id name has function name